src/collective/easyform/browser/widgets.zcml

```diff
@@ -74,6 +74,24 @@
       class=".widgets.RichLabelRenderWidget"
       layer="..interfaces.IEasyFormLayer"
       />
+  <browser:page
+      name="ploneform-render-widget"
+      for="z3c.form.interfaces.IWidget"
+      layer="..interfaces.IEasyFormLayer"
+      class=".widgets.RenderWidget"
+      permission="zope.Public"
+      />
```

Comment:
I do not think, easyform should override all standard z3cform widgets site wide. This should only match for `IEasyFormForm` widgets